server/server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+
+const app = express();
+app.get('/api/customers', (req, res) => {
```

Comment:
as the app gets bigger you gonna have to split the routes ( `app,get` and stuff) into a different folder and files ( as in move them into the controllers folder ) do you want do it now or when you have lots of routes?

Reply:
Whenever needed is better

Reply:
cool